tests/foreman/cli/test_contentview.py

```diff
@@ -5320,7 +5320,7 @@ def test_positive_arbitrary_file_repo_promotion(self):
             1. Promote the CV to the Environment
 
         :expectedresults: Check arbitrary files from FR is available on
-            environment
+            content view version environment
```

Comment:
I believe
`content view version` without `environment`

Reply:
As per the test step, we have to check the repository avaiability in published content-view's particular version having specific environment.
So I think it is correct.